configuration/backends/smatch/smatch-wrapper.sh

```diff
@@ -114,8 +114,11 @@ function run_smatch
   logwrapper "${TOOLPREFIX}"SMATCH"${TOOLSUFFIX}" "smatch $SMATCH_EXTRA_ARG $@"
   # in case the source file is analyzed multiple times, collect all messages
   SMATCH_STATUS=0
-  smatch $SMATCH_EXTRA_ARG "$@" "${source_files[@]}" "${header_files[@]}" &>> "$rf" || SMATCH_STATUS=$?
 
+  # do detour via cgcc
```

Comment:
Randomly placing it here - from the commit log: "This way, tooling
inside smatch is used as well" -- what sort of "tooling" is this, or really, what does this mean? Would you mind rewording a bit?